community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java

```diff
@@ -820,6 +820,18 @@
     public static final Setting<Duration> bookmark_ready_timeout = buildSetting(
             "dbms.transaction.bookmark_ready_timeout", DURATION, "30s" ).constraint( min( Duration.ofSeconds( 1 ) ) ).build();
 
+    public enum TransactionStateMemoryAllocation
+    {
+        ON_HEAP,
+        OFF_HEAP
+    }
+
+    @Description( "Defines where to allocate memory for storing transaction state. Valid values are `on_heap` and `off_heap`." )
```

Comment:
lets not hardcode constant names in a description, this gives to future maintainer huge possibility to not update this description in case if options will be updated

Reply:
✅